README.md

```diff
@@ -1,11 +1,21 @@
 Infrastructure Management for taskcluster
 =========================================
 
+To set up your local working environment run:
```

Comment:
If you feel like combining this with the `to run` steps since it is a bit confusing that they are similar please do, but otherwise just land!